static/js/components/FollowupRequestLists.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+
+import * as Actions from "../ducks/source";
+import styles from "./FollowupRequestLists.css";
```

Comment:
makeStyles

Reply:
@stefanv  I think I have isolated the issue here, has to do with groups.all not being loaded at `FollowupRequestForm` render time, adding a guard for this.

Reply:
fixed